app/assets/javascripts/components/common/feedback.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import OnClickOutside from 'react-onclickoutside';
+
+const Feedback = React.createClass({
+  displayName: 'Feedback',
+
+  getInitialState() {
+    return {
+      show: false
+    };
+  },
+
+  show() {
+    this.setState({ show: true });
+  },
+
+  hide() {
+    this.setState({ show: false });
+  },
+
+  handleClickOutside() {
+    this.hide();
+  },
+
+  render() {
+    let button;
+    if (this.state.show) {
+      button = <button onClick={this.hide} className="button dark small">Okay</button>;
+    } else {
+      button = <a onClick={this.show} className="button dark small">{I18n.t('courses.feedback')}</a>;
+    }
+
+    let modal;
+    if (!this.state.show) {
+      modal = <div className="empty" />;
+    } else {
+      modal = (
+        <div className="article-viewer feedback">
+          <h2>Feedback</h2>
+          <a className="button small diff-viewer-feedback" href="" target="_blank">{I18n.t('courses.suggestions_feedback')}</a>
```

Comment:
Styling: The margin-right property of diff-viewer-feedback class must be set to 0, So as to align it with the Okay button.